src/startupscriptgenerator/src/php/main.go

```diff
@@ -14,19 +14,26 @@ func main() {
 	common.PrintVersionInfo()
 
 	appPathPtr := flag.String("appPath", ".", "The path to the application folder, e.g. '/home/site/wwwroot/'.")
+	manifestFileDirPathPtr := flag.String(
+		"manifestFileDirPath",
```

Comment:
this flag is essentially the same across generators - the `flag.String` can moved into our `common` package and used by them. this will decrease repetition and make it easier to change this flag down the road

Reply:
Hmm, I will see how this can be achieved, but I agree that this is becoming repetitive and is error-prone